diff -up kosmindoormap-22.12.3/src/CMakeLists.txt.omv~ kosmindoormap-22.12.3/src/CMakeLists.txt
diff -up kosmindoormap-22.12.3/src/osm/CMakeLists.txt.omv~ kosmindoormap-22.12.3/src/osm/CMakeLists.txt
--- kosmindoormap-22.12.3/src/osm/CMakeLists.txt.omv~	2023-03-08 23:49:03.641133176 +0100
+++ kosmindoormap-22.12.3/src/osm/CMakeLists.txt	2023-03-08 23:51:42.107849666 +0100
@@ -7,7 +7,7 @@ if (Protobuf_FOUND AND Protobuf_PROTOC_E
         pbf/fileformat.proto
         pbf/osmformat.proto
     )
-    set(EXTRA_LIBS ${Protobuf_LIBRARIES} ZLIB::ZLIB)
+    set(EXTRA_LIBS ${Protobuf_LIBRARIES} ZLIB::ZLIB -labsl_log_internal_check_op -labsl_log_internal_message)
     set_source_files_properties(${pbf_srcs} ${pbf_hdrs} PROPERTIES SKIP_AUTOMOC ON)
 endif()
 
